packages/infra/lib/api-stack/api-service.ts

```diff
@@ -180,12 +180,12 @@ export function createAPIService({
   semanticSearchEndpoint: string | undefined;
   semanticSearchIndexName: string | undefined;
   semanticSearchModelId: string | undefined;
-  surescriptsReplicaBucket: s3.IBucket;
-  medicationBundleBucket: s3.IBucket;
-  surescriptsSynchronizeSftpQueue: IQueue;
-  surescriptsSendPatientRequestQueue: IQueue;
-  surescriptsReceiveVerificationResponseQueue: IQueue;
-  surescriptsReceiveFlatFileResponseQueue: IQueue;
+  surescriptsReplicaBucket?: s3.IBucket;
+  medicationBundleBucket?: s3.IBucket;
+  surescriptsSynchronizeSftpQueue?: IQueue;
+  surescriptsSendPatientRequestQueue?: IQueue;
+  surescriptsReceiveVerificationResponseQueue?: IQueue;
+  surescriptsReceiveFlatFileResponseQueue?: IQueue;
```

Comment:
Since we're touching this, can we make a single param for SS configs? The list is completely out of hand, more than 100 props now according to 🐰

Reply:
This also helps, because we can make this the same type of the output of the SS nested stack, and so we wouldn't need to have all those props popping out in the middle of the API stack.

That sounds like a good improvement on the current pattern!